lib/src/cli.rs

```diff
@@ -111,10 +111,6 @@ pub(crate) struct SwitchOpts {
     #[clap(long)]
     pub(crate) enforce_container_sigpolicy: bool,
 
-    /// Enable verification via an ostree remote
-    #[clap(long)]
-    pub(crate) ostree_remote: Option<String>,
```

Comment:
Technically we claimed public CLI/API stability but I can't find any users of this on public source code, and I can't imagine there are any private ones.

To be clear: Everyone should use container signing, not ostree commit signatures.